src/main/java/games/strategy/triplea/ai/proAI/ProAi.java

```diff
@@ -58,7 +58,7 @@
 public class ProAi extends AbstractAi {
 
   // Odds calculator
-  private static final IOddsCalculator concurrentCalc = new ConcurrentOddsCalculator("ProAi");
+  private static final IOddsCalculator concurrentCalc = new OddsCalculator(null);
```

Comment:
Can we avoid the `null`? Seems like the burden of knowing that the param can be null should not lie with the client, it ties implementation details to usage.

Reply:
Doesn't look like it :/